080-Recipes/010-csv-data/010-import-csv-data-UI.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Import CSV data
+navTitle: Import CSV data
+keywords: ['csv', 'import', 'migrate data']
+description: Import records from a local CSV file into a Xata database table
+slug: recipes/import-data
+published: true
+---
+
+## Overview
+
+A CSV file is a standard text file where data fields are typically divided by commas (,), and saved with the `.csv` extension.
+You can import and work with CSV file data in your Xata database.
+
+**NOTE:** While CSV files are explicity mentioned, Xata supports the import of various file formats, including tab-separated values (TSV), comma-separated values (CSV), delimiter-separated values (DSV), and plain text data. This covers a range of extensions including `.tsv`, `.csv`, `.dsv`, and `.txt` formats.
+
+You can import CSVs using the following methods:
+
+- Import a CSV file using the Xata UI (recommeneded)
+- Import a CSV file using the Xata CLI
+
+### Considerations
+
+- Decide if you are using the Xata UI (Workspace) or CLI to import CSV files. Using the UI is the recommended option, but you can use both to edit records after importing.
+- Ensure your file's accuracy before importing. Certain values may become uneditable after importing. For errors in the imported file, you'll need to delete, edit the CSV file, and try importing again.
```

Comment:
> Certain values may become uneditable after importing

The only thing this is true for is ids for most values you can edit them in the UI